tools/fork-network/src/cli.rs

```diff
@@ -201,7 +200,6 @@ struct ResetCmd;
 struct Validator {
     account_id: AccountId,
     public_key: PublicKey,
-    #[serde(with = "dec_format")]
```

Comment:
I wonder if we can also remove `dec_format` or is it still used somewhere?

Reply:
Yes, there are total 5 usages. Including timestamp_nanosec which is `u64` and gas fields including `u128` [here](https://github.com/near/nearcore/blob/050f08f063bad5517499be2f5eb343ff1aa06955/core/primitives/src/views.rs#L2720) and `Gas` type